api/sheets/get_property.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Get Node Property
+nav_order: 7
+parent: Google Sheets
+grand_parent: API
+---
+
+# Get the values of a given property for a node.
+
+## =DCPROPERTY(dcids, property)
+
+Given a property DCID, together with a string DCID, a row of cells of DCIDs, or a column of cells of DCIDs, get the values of the given property for those DCIDs.
```

Comment:
also why emphasize "string DCID"? string does not quite correspond to "row" or "column"

Reply:
The three ways of passing DCIDs look like

- DCPROPERTY("geoId/06", "typeOf")
- DCPROPERTY(A1:A2, "typeOf")
- DCPROPERTY(A1:B1, "typeOf")

I want to make it clear that all 3 are options.